### PR TITLE
Wgsmetricsalgo

### PR DESCRIPTION
**Pull request by Gwennid**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] Make sure your code lints (`nf-core lint`).

This PR adds a local module for Sentieon's WgsMetricsAlgo (which is similar to Picard's CollectWgsMetrics). I first wrote and tested it in nf-core/modules repo (using Conda to run Sentieon with Clinical Genomics Gothenburg license) and it worked well. I copied the code to use Sentieon's "secret" and stub feature from other Sentieon local modules.

### CODE REVIEW

File: modules/local/sentieon/wgsmetricsalgo.nf [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    tuple val(meta), path('*wgs_metrics.txt'), emit: wgs_metrics
+    path  "versions.yml"                     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args ?: ''
+    def input  = bam.sort().collect{"-i $it"}.join(' ')
+    def prefix       = task.ext.prefix ?: "${meta.id}"
```
Comment (ramprasadn) [inline modules/local/sentieon/wgsmetricsalgo.nf:23]:
```suggestion
    def prefix  = task.ext.prefix ?: "${meta.id}"
```

File: modules/local/sentieon/wgsmetricsalgo.nf [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        --algo WgsMetricsAlgo ${prefix}_wgs_metrics.txt \\
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        sentieon: \$(echo \$(sentieon driver --version 2>&1) | sed -e "s/sentieon-genomics-//g")
+    END_VERSIONS
+    """
+
+    stub:
+    def prefix       = task.ext.prefix ?: "${meta.id}"
```
Comment (ramprasadn) [inline modules/local/sentieon/wgsmetricsalgo.nf:45]:
```suggestion
    def prefix = task.ext.prefix ?: "${meta.id}"
```

**Review by ramprasadn — COMMENTED** [review]
Nice work, @Gwennid! Just a couple of minor suggestions 😄

**Review by sima-r — APPROVED** [review]
Will you add an issue to the project bord regarding integrating this module in the workflow?

**Pull request merged**